main.css

```diff
@@ -1,370 +0,0 @@
-  grid-area: aliens; 
-  position: relative;
-  justify-content: space-around;
-  align-items: flex-start;
-  display: flex;
-  flex-direction: column;
-  height: 50%;
-}
-#right-content{
-  grid-area: right-content;
-  display: grid;
-  grid-template-rows: 1fr 1fr;
-  grid-template-columns: 1fr;
-  grid-template-areas: 
-  "main-nav-right"
-  "of-note";
-}
-.post-content {
-  border-bottom: solid grey 1px;
-  margin-bottom: 30px;
-  padding-bottom: 30px;
-}
-#main-nav-right {
-  display: flex;
-  flex-direction: column;
-  justify-content: space-around;
-}
-.post{
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-}
-.post-head{
-  padding-top: 20px;
-  padding-bottom: 20px;
-}
-#note-chunk {
-  display: flex;
-}
-#of-note {
-  display: flex;
-  flex-direction: column;
-  }
-.note-text {
-  margin-bottom: 15px;
-}
-#note-box{
-  background-color: blueviolet;
-  width: 80px;
-  height: 80px;
-  float: left;
-  margin-right: 15px;
-  margin-bottom: 15px;
-}
-
-/*----"Learn More" CSS----*/
-
-#learn-main {
-  width: 75%;
-  display: grid;
-  grid-template-columns: 1fr 2fr;
-  grid-template-rows: 1fr;
-  grid-template-areas: 
-    "right-content left-content";
-  margin-left: auto;
-  margin-right: auto;
-  grid-gap: 20px;
-}
-/*----Post-Only CSS----*/
-#main-only{
-  display: flex;
-  justify-content: center;
-}
-#content-only{
-  width: 75%;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-}
-#head-only{
-  padding-bottom: 30px;
-}
-#pic-only{
-  padding-bottom: 30px;
-}
-/* - - - main content end - - - - */ 
-
-/*  - - - - footer  - - - - */
-#footer {
-  background-color: #d2d2d4;
-  position: static;
-  width: auto;
-  height: 350px;
-  margin-top: 30px;
-}
-#get-in-touch {
-  text-align: center;
-  padding-top: 10px;
-}
-#grid-for-footer {
-  display: grid;
-  grid-template-columns: 1fr 1fr;
-  grid-template-rows: minmax(275px) 1fr 1fr 1fr 1fr;
-  grid-template-areas: 
-    "form icons"
-     "form icons"
-     "form icons"
-     "form icons";
-  padding-right: 100px;
-  padding-left: 100px;
-}
-#form {
-  display: grid;
-  margin: 10px;
-  font-family: Roboto, Courier;
-}
-#comments {
-  height: 50px;
-  width: auto;
-  margin-bottom: 10px;
-}
-#icons {
-  display: grid;
-  grid-template-columns: 1fr 1fr;
-  grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
-  grid-template-areas: 
-  "footer-text footer-text"
-  "icon-flex icon-flex"
-  "icon-flex icon-flex"
-  "icon-flex icon-flex"
-  "icon-flex icon-flex";
-}
-#footer-text {
-  grid-area: 1 / span 2;
-}
-.icon-flex{
-  display: flex;
-  justify-content: flex-start;
-  overflow-wrap: break-word;
-  align-items: center;
-  padding-right: 15px;
-  padding-left: 15px;
-}
-h3 { /* <h3> reserved for icon text */
-  font-size: 10px;
-  font-family: Roboto, Courier;
-  text-align: left;
-}
-.icon {
-  width: 30px;
-  height: 30px
-}
-/*  - - - - mobile footer  - - - - */
-@media screen and (max-width: 500px) {
```

Comment:
Great use of a media query here to change the layout based on the screen width!!